.github/CODEOWNERS

```diff
@@ -13,6 +13,7 @@
 
 # Service teams
 /sdk/appconfiguration/                              @mssfang @alzimmermsft @conniey @sima-zhu
+/sdk/appconfiguration/spring-cloud-azure-*          @mrm9084 @saragluna @yiliuTo @chenrujun @zhoufenqin @jialindai
```

Comment:
@danieljurek Correct me if I'm wrong but I suspect this might cause issues with your pipeline notification work. I suspect that the notifications for the shared ci.yml pipelines will only go to the set of folks listed as owners under sdk/appconfiguration.

Reply:
This won't cause the notification configuration system to crash but it may not result in perfect routing of notifications. 

Notifications are configured based on entries in the CODEOWNERS file that map to the location of the pipeline's yaml file. Java pipelines are configured at the level of the service (not package) so the yaml file is located in `sdk/appconfiguration` and not anywhere deeper like `sdk/appconfiguration/spring-cloud-azure-foo/`. If there is a failure of any sort in the appconfiguration pipeline then the notifications will be directed to those entries in CODEOWNERS whose path matches the beginning of the `ci.yml` or `tests.yml` file (i.e. `sdk/appconfiguration/` and not `/sdk/appconfiguration/another-folder-deeper-in`). 

We also don't have an easy way to notify for failures at the level of the package. If a failure happens in `azure-data-appconfiguration` or `spring-cloud-azure-appconfiguration-web` we currently only support notifying everyone on the `/sdk/appconfiguration/` line. 

As the system is set up today there are two ways to receive notifications for all failures for a given service: 

* Add your entries to the `/sdk/appconfiguration/` line or
* Manually subscribe yourself to the parent notification group in Azure DevOps. In this case it'd be [`java - appconfiguration - tests -- Notifications`](https://dev.azure.com/azure-sdk/internal/_settings/teams?subjectDescriptor=vssgp.Uy0xLTktMTU1MTM3NDI0NS03MjEyMjY4NDEtNDc1NTgxMjYxLTI3MDQxNzIzOTItMzQ2Mzk5NTc2NS0xLTI5OTIyOTY0NjctNDA1MTI5ODYzNC0zMDA0OTkxMTkzLTE3MTc5NDUyOTE) and/or [`java - appconfiguration --Notifications`](https://dev.azure.com/azure-sdk/internal/_settings/teams?subjectDescriptor=vssgp.Uy0xLTktMTU1MTM3NDI0NS03MjEyMjY4NDEtNDc1NTgxMjYxLTI3MDQxNzIzOTItMzQ2Mzk5NTc2NS0xLTM4Njk3MjMyODUtNTI4NDg1MTg1LTMxNzE1NjUxMjEtMjQ3NDAwMTgwOA)

Reply:
Hi, @weshaggard , @danieljurek .

Thank you for your check.

> Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.

I Just want to set the reviewers. I do not need to subscribe the pipeline notification. So I'll not modify any more.

Refs: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#about-code-owners

Reply:
>I do not need to subscribe the pipeline notification.

As you guys start getting your tests running you should definitely subscribe to the pipeline notifications so you can ensure that your tests stay green.